src/js/_store/modules/categories.js

```diff
@@ -20,6 +26,29 @@ export default {
       state.errorMessage = message;
       state.loading = false;
     },
+    toggleLoading(state) {
+      state.loading = !state.loading;
+    },
+    clearMessage(state) {
+      state.doneMessage = '';
+      state.errorMessage = '';
+    },
+    // doneMessageにinputに入力した値をカテゴリー名に入れる
+    displayDoneMessage(state, payload = { message: '成功しました' }) {
+      state.doneMessage = payload.message;
+    },
+    updateValue(state, payload) {
+      state.targetCategory = { ...state.targetCategory, name: payload.name };
+    },
+    initPostCategory(state) {
+      state.targetCategory = {
+        id: null,
+        name: '',
+      };// targetCategoryを初期化
+    },
```

Comment:
[imo]

`Post`とついていると、作成の時だけしか使用できないメソッドなのかな？と他の開発者からしたら思ってしまう可能性もあります。
そうではなく、あくまで「対象のカテゴリ（targetCategory）」を初期化しているので以下のようなミューテーション名が適切かと思いました。

```suggestion
    initTargetCategory(state) {
      state.targetCategory = {
        id: null,
        name: '',
      };// targetCategoryを初期化
    },
```

Reply:
63b51786231f95e82c6251f02464fd8acb0bc917
いただいたミューテーション名に修正いたしました。